lib/pbxProject.js

```diff
@@ -1644,6 +1644,10 @@ function pbxShellScriptBuildPhaseObj(obj, options, phaseName) {
     obj.shellPath = options.shellPath;
     obj.shellScript = '"' + options.shellScript.replace(/"/g, '\\"') + '"';
 
+    if (options.alwaysOutOfDate != null) {
```

Comment:
The current codebase uses the triple equality and we should continue this pattern for consistency.

This will more strictly test against `null` without type coercion (though functionality I believe in this case it isn't any different than `!=`).

```suggestion
    if (options.alwaysOutOfDate !== null) {
```